src/main/java/am/ik/yavi/constraint/base/NumericConstraintBase.java

```diff
@@ -18,10 +18,7 @@
 import java.util.function.Predicate;
 
 import static am.ik.yavi.core.NullAs.VALID;
-import static am.ik.yavi.core.ViolationMessage.Default.NUMERIC_GREATER_THAN;
-import static am.ik.yavi.core.ViolationMessage.Default.NUMERIC_GREATER_THAN_OR_EQUAL;
-import static am.ik.yavi.core.ViolationMessage.Default.NUMERIC_LESS_THAN;
-import static am.ik.yavi.core.ViolationMessage.Default.NUMERIC_LESS_THAN_OR_EQUAL;
+import static am.ik.yavi.core.ViolationMessage.Default.*;
```

Comment:
Can you run `mvn formatter:format` ?

Reply:
Yes, forgot about that! 


Would be nice to have a contribution guideline so we know what should be done before sending a PR.